man/plot_sankey.Rd

```diff
@@ -12,6 +13,15 @@ plot_sankey(
   png_name = "sankey.png",
   nodes_order_from_data = FALSE
 )
+
+prep_sankey(
+  data_alignment,
+  region,
+  year,
+  group_var,
+  middle_node,
+  middle_node2 = NULL
+)
```

Comment:
Style: It seems a bit counter-intuitive that `prep_*` comes after `plot_*` in the documentation here. What do you think?

Reply:
I did that because in my mind the `plot_*` function is the "real" function, or the most important function, or at least the primary function that we most want to talk about, and the `prep_*` function is just a helper for it?

Reply:
If it's intentional then makes sense! just wanted to make sure it wasn't accidental.

Although curious what @jacobvjk thinks too?

Reply:
tbh I think it doesn't really matter which comes first. If we do not export these functions, there is no "primary" function for me. In a sense, if we decide at some point that we do not want to export these functions at all, we may even scrap the manual for the prep_* function, no?
I think for now it makes sense to keep both, because we may want to export them again at some point and they are basically ready for that. long story short: fine for me